lib/beacon/live_admin/live/page_editor_live/form_component.ex

```diff
@@ -1,19 +1,18 @@
 defmodule Beacon.LiveAdmin.PageEditorLive.FormComponent do
-  require IEx
-  require Logger
   use Beacon.LiveAdmin.Web, :live_component
 
   alias Beacon.LiveAdmin.Config
   alias Beacon.LiveAdmin.Content
+  alias Beacon.LiveAdmin.WebAPI
 
   @impl true
-  def update(
-        %{site: site, page: page, visual_mode: visual_mode, components: components} = assigns,
-        socket
-      ) do
+  def update(%{site: site, page: page} = assigns, socket) do
+    # FIXME: new pages
```

Comment:
@leandrocp what did you mean by this?

Reply:
New pages have no path so `WebAPI.Page.show/1` fails. Need to change that `data` function to consider this scenario